source/_components/hive.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: "Hive"
+description: "Instructions on how to integrate Hive devices with Home Assistant."
+date: 2017-09-24 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hive.png
+ha_category: Hub
+ha_release: 0.62
+ha_iot_class: "Cloud Polling"
+---
+
+
+This Hive component is the main component to setup and integrate all supported Hive devices.  Once configured with the minimum required details it will detect and add all your Hive devices in to Home Assistant, including support for multizone heating.
```

Comment:
Double spaces detected: `Hive devices.  Once`, remove one.
Typo: `setup` -> `set up`
Typo: `in to`-> `into`

Reply:
Updated